src/appleseed/renderer/modeling/environmentedf/oslenvironmentedf.cpp

```diff
@@ -76,11 +142,21 @@ namespace
     {
       public:
         OSLEnvironmentEDF(
-            const char*             name,
-            const ParamArray&       params)
+            const char*         name,
+            const ParamArray&   params)
           : EnvironmentEDF(name, params)
+          , m_importance_map_width(0)
+          , m_importance_map_height(0)
+          , m_probability_scale(0.0f)
         {
             m_inputs.declare("osl_background", InputFormatEntity, "");
+
+            // TODO:
+            // Add optional values from m_inputs (see LatLongMapEnvironmentEDF).
+            m_phi_shift = deg_to_rad(m_params.get_optional<float>("horizontal_shift", 0.0f));
```

Comment:
Rotating the environment is something that can be easily done in an OSL shader if needed and that is the place where it should go. I prefer that we don't handle it here.